Frontend/src/main/java/com/harmonia/HarmoniaApplication.java

```diff
@@ -13,7 +13,7 @@
 /**
  * Class representing the running class of the Harmonia application.
  *
- * @author OTP1 Group 3: Johanna Toivanen, Mohammed Al-Jewari, Sampo Savolainen, Jesper Ojala
```

Comment:
Harmonia Team :frog:

Reply:
magic, I changed it before you left your comment.